src/commands/info/open-ticket.ts

```diff
@@ -0,0 +1,16 @@
+import { CommandInteraction } from 'discord.js';
+
+require('dotenv').config();
+require('discord.js');
+
+module.exports = {
+  /* @param {Client} client
+   * @param {Interaction} interaction */
+  callback: (interaction: CommandInteraction) => {
+    console.log('Bug/Feature Link');
+    interaction.reply(process.env.SOURCE + '/issues');
```

Comment:
I get the same error as [`source.ts`](https://github.com/stinkyfi/MaddenBot/pull/11/files#r1240613450)

```sh
There was an error running this command: TypeError: interaction.reply is not a function
```

Reply:
Yes it seems to be my bad for not being capable of defining interaction type properly, it seems that client is `client` but interaction is not `interaction` type but something else, I also tried asigning it to `any` but it does not seem to solve the problem. I am sorry you have to deal with my mistakes.

Reply:
I am going to propose removing the `/info` commands for now. I think your typescript changes are highly useful.
I can't make sense of the error right now, and worst case they can be reworked to embeded message.

actively testing the other commands

Reply:
Sorry for the dellay kind stinkyfi, I have tried some stuff now that my brain SORT of works and heres a code snippet for you to see : 
``` console.log('Get Source');
    if (interaction instanceof CommandInteraction) {
      console.log('interaction is a CommandInteraction');
    } else {
      console.log('interaction is not a CommandInteraction');
      console.log(typeof interaction);
    }
```
As for the output :
```
Connected to DB
Testbot2#5223 is online.
Get Source
interaction is not a CommandInteraction
object
There was an error running this command: TypeError: interaction.reply is not a function
``` 
The problem is not with how you wrote the code, like I have said previously the problem was either the import or the type of for interaction and as I have just showed you it seems that in this case interaction is not type of CommandInteraction, rather than discard the command all together you should keep it, I will try to find the proper type ( neither object or any work so ugh ).

Reply:
yeah, it's a tough one. I would only suggest deleting them temporarily to get this merge request in. Creating new tickets to address them in the future. They aren't critical features but more nice to haves. Converting them to use discord.js embeded messages might be a simple enough solution as well.

Reply:
Fixed, no need to get rid of the file, the problem was interaction was running but it had nowhere to run therefore it didn't count as a function. Here is my solution :
```
import { CommandInteraction, Client } from 'discord.js';
//make sure to also import it
callback: (_client: Client, interaction: CommandInteraction)
```

Reply:
Well done! please see my `.eslintrc.json` changes, let's get the linter working and we can merge this in.